test/cmdlineTests/model_checker_show_unproved_false_chc/args

```diff
@@ -1 +0,0 @@
---model-checker-engine chc --model-checker-show-unproved false
```

Comment:
In theory, the test can stay after removing `--model-checker-show-unproved false`. But fine either way.

Reply:
But there's already a test with that

Reply:
(without the option completely)